README.md

```diff
@@ -127,6 +127,31 @@ The *getRange()* method returns the currently-set measurement range of the senso
 server.log(format("Current Sensor Range is +/- %dG", accel.getRange()));
 ```
 
+### configureFifoInterrupt(*state[, fifomode][, watermark]*)
+
+This method configures an interrupt when the FIFO buffer reaches the set watermark:
+
+| Parameter | Type | Default Value | Description |
+| --------- | ---- | ------------- | ----------- |
+| *state* | Boolean | N/A | `true` to enable, `false` to disable |
+| *fifomode* | bitfield | *FIFO_STREAM_MODE* | See table below |
+| *watermark* | Integer | 28 | Number of buffer slots filled to generate<br> interrupt (buffer has 32 slots) |
+
+```squirrel
+// Configure the FIFO buffer in Stream Mode and set interrupt generator to
+// generate an interrupt when there are 30 entries in the buffer
+accel.configureFifoInterrupt(true, FIFO_STREAM_MODE, 30);
```

Comment:
A quick example of how to configure the interrupt pin, read the FIFO and how to react to overflow events would be really helpful.